model/shared/src/main/scala/explore/model/OdbItcResult.scala

```diff
@@ -33,7 +34,10 @@ object OdbItcResult {
   ) extends OdbItcResult
       derives Eq {
     def toItcExposureTime: ItcExposureTime =
-      ItcExposureTime(OverridenExposureTime.FromItc, exposureTime, exposures)
+      ItcExposureTime(OverridenExposureTime.FromItc,
+                      exposureTime,
+                      PosInt.unsafeFrom(exposures.value)
```

Comment:
How do you know this is OK?

Reply:
We logically know it is never 0
The next lucuma-core will reconcile the mismatching usages across PosInt and NonNegInt